strawberryfields/apps/graph/resize.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+r"""
+Graph resizing
```

Comment:
You may have to remind me, but is this the module that is being renamed to 'modify'?